### PR TITLE
add getter/setter for a member's receipt contact

### PR DESCRIPTION
**Pull request by gianluca-token**
add receiptRequested setter on TransferTokenBuilder

### CODE REVIEW

File: src/http/AuthHttpClient.js [hunk 1]
```diff
@@ -434,6 +434,40 @@ class AuthHttpClient {
...
+     * @return {Object} response - response to the API call
+     */
+    async setReceiptContact(id, contact) {
+        const req = {
+            memberId: id,
+            contact: contact
+        };
+        const request = {
+            method: 'put',
+            url: `/profilepicture`,
```
Comment (colin-token) [inline src/http/AuthHttpClient.js]: why is this putting to profile picture?

File: src/http/AuthHttpClient.js [hunk 2]
```diff
@@ -434,6 +434,40 @@ class AuthHttpClient {
...
+    /**
+     * Replaces member's receipt contact.
+     *
+     * @param {string} id - member ID of member whose receipt contact we will set
+     * @param {Object} contact - receipt contact to set
+     * @return {Object} response - response to the API call
+     */
+    async setReceiptContact(id, contact) {
+        const req = {
+            memberId: id,
```
Comment (colin-token) [inline src/http/AuthHttpClient.js]: why is it not the member id of the member who is setting it?

File: src/http/AuthHttpClient.js [hunk 3]
```diff
@@ -434,6 +434,40 @@ class AuthHttpClient {
...
+    /**
+     * Gets a member's receipt contact.
+     *
+     * @param {string} id - member ID of member whose receipt contact we want
+     * @return {Object} response - response to the API call
+     */
+    async getReceiptContact(id) {
+        const request = {
+            method: 'get',
+            url: '/members/${id}/receipt-contact'
```
Comment (colin-token) [inline src/http/AuthHttpClient.js]: are we letting members fetch other members' receipt contacts?

**Comment by zixunjian** [comment]
you need to bump version in package.json

File: src/http/AuthHttpClient.js [hunk 4]
```diff
@@ -434,6 +434,38 @@ class AuthHttpClient {
         return this._instance(request);
     }
 
+    /**
+     * Replaces member's receipt contact.
+     *
+     * @param {Object} contact - receipt contact to set
```
Comment (colin-token) [inline src/http/AuthHttpClient.js]: what is the object type? is it a Contact object? or is it just going to be a string?

**Comment by colin-token** [comment]
Also need to add sdk-java-tests after this is merged

File: src/http/AuthHttpClient.js (hunk 4 above)
Comment (gianluca-token) [reply src/http/AuthHttpClient.js]: Contact object. Should I annotate as {Contact}? I notice other calls using {Object} for custom objects.

**Pull request merged**